### PR TITLE
Add benchmark for as http

### PR DESCRIPTION
**Pull request by anson627**
raw-http:
benchmark result { totalRequests: 5000,
  totalErrors: 0,
  totalTimeSeconds: 0.865796189,
  rps: 5775,
  meanLatencyMs: 0,
  maxLatencyMs: 8,
  percentiles: { '50': 1, '90': 1, '95': 1, '99': 4 },
  errorCodes: {} }

as-http:
benchmark result { totalRequests: 5000,
  totalErrors: 0,
  totalTimeSeconds: 2.087116424,
  rps: 2396,
  meanLatencyMs: 0,
  maxLatencyMs: 34,
  percentiles: { '50': 1, '90': 2, '95': 3, '99': 8 },
  errorCodes: {} }

r: @ShanniLi @Raynos @jcorbin

### CODE REVIEW

File: node/examples/lbpool_ingress.js [hunk 1]
```diff
@@ -1,100 +0,0 @@
-// Copyright (c) 2015 Uber Technologies, Inc.
```
Comment (Raynos) [inline node/examples/lbpool_ingress.js]: Why was this deleted ?

File: node/benchmarks/http/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    if (self.opts.relay) {
+        targetPort = self.ports.egressPort;
+    }
+    var options = {
+        url: 'http://127.0.0.1:' + targetPort,
+        concurrency: self.opts.numClients,
+        maxRequests: self.opts.numRequests,
+        agentKeepAlive: self.opts.keepAlive
+    };
+    loadtest.loadTest(options, function(error, result) {
```
Comment (Raynos) [inline node/benchmarks/http/index.js]: This should be a child process ideally.

**Comment by Raynos** [comment]
Drain test is flappy :(

**Comment by Raynos** [comment]
@anson627 See if you can fix that flappy drain test :) would be cool.

Otherwise this looks good to me.

File: node/examples/lbpool_ingress.js (hunk 1 above)
Comment (anson627) [inline node/examples/lbpool_ingress.js]: adding streamed parameter to http_ingress.js makes this duplicate

Comment (Raynos) [inline node/examples/lbpool_ingress.js]: cool.
